scenarios/goldilocks.benchmarks.yml

```diff
@@ -37,9 +48,38 @@ scenarios:
 
   stage1vanilla:
     application:
-      job: aspnetbenchmarks
+      job: stage1aspnetbenchmarks
     load:
       job: wrk
       variables:
         presetHeaders: json
-        path: /todos
\ No newline at end of file
+        path: /todos
+
+  stage1grpcpublishaot:
+    application:
+      job: stage1grpcaspnetbenchmarks
+      buildArguments: 
+        - "/p:PublishAot=true"
+        - "/p:StripSymbols=true"
+    load:
+      job: h2LoadClient
+      variables:
+        presetHeaders: grpc
+        path: /goldilocks.TodoService/GetAllTodos
+        body: AAAAAAA=
```

Comment:
What does this do?

Reply:
Base64 encoded POST body of the HTTP request. It's an empty gRPC message, which is 5 zeros.